src/__tests__/findDOMNode.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import { render, unmountComponentAtNode, findDOMNode } from 'react-dom-lite';
+import * as TestUtils from 'react-dom-lite/test-utils';
```

Comment:
Why not use (React's test-utils)[https://reactjs.org/docs/test-utils.html]?

Reply:
That would use react-dom to render, which would end up not testing the right things. For some of the other methods it might work (like the traversal ones) but we haven't had a need for that yet. Ultimately I'd like to write an enzyme adapter to ensure that works, given it's the more popular testing tools